op-acceptance-tests/tests/jovian/calldata_footprint_test.go

```diff
@@ -0,0 +1,103 @@
+package jovian
+
+import (
+	"crypto/rand"
+	"fmt"
+	"math/big"
+	"sync"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/txplan"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/stretchr/testify/require"
+)
+
+func sendTx(t devtest.T, eoa *dsl.EOA, to common.Address, calldataSize int) {
+	data := make([]byte, calldataSize)
+	_, err := rand.Read(data)
+	t.Require().NoError(err)
+	eoa.Transact(
+		eoa.Plan(),
+		txplan.WithTo(&to),
+		txplan.WithData(data),
+	)
+	fmt.Println("sent tx", eoa.Address())
+}
+
+// TestCalldataFootprint generates a large volume of transactions with configurable calldata size.
+// It is used to test the calldata footprint response of the L2 under Jovian HF.
+func TestCalldataFootprint(gt *testing.T) {
+	t := devtest.SerialT(gt)
+	sys := presets.NewMinimal(t)
+
+	// check that the chain is running on Jovian fork
+	err := dsl.RequiresL2Fork(t.Ctx(), sys, 0, rollup.Jovian)
+	require.NoError(t, err, "Jovian fork must be active for this test")
+
+	calldataSize := 120_000
+	eoaGroupSize := 50
+	daFactor := params.CalldataFootprintCost
+
+	// create eoas
+	eoaGroup := make([]*dsl.EOA, eoaGroupSize)
+	for i := range eoaGroup {
+		eoaGroup[i] = sys.FunderL2.NewFundedEOA(eth.HundredEther)
+	}
+
+	// all eoas send tx to the same recipient simultaneously
+	to := eoaGroup[0].Address()
+	wg := sync.WaitGroup{}
+	for i := range eoaGroup {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			sendTx(t, eoaGroup[i], to, calldataSize)
```

Comment:
Right now the mere presence of transactions, even with negligible calldata, is causing bad blocks and failure to include the tx. Not sure why, investigating this, but I assume it's an issue with the feature.